### DISCUSSION TITLE
Add to the Accrescent.app store

### DISCUSSION THREAD

**Discussion by JaByteOS** [ente-io/ente#2504] [Enhancements]
Have you considered or are you planning on putting your app on Accrescent (app store focused on security, privacy, and usability) 

Accrescent is a private and secure Android app store built with modern features in mind.

It was just endorsed by GrapheneOS (considered by many to be the most private and secure mobile operating). GrapheneOS also now has accrescent in their app store. 

Right now there is only 9 apps on it. So any new apps added get great installation opportunities from the privacy and security minded community. 

Anyway, I really like your app, and would love to see it on Accrescent because it's so easy to install and update apps compared with f-droid, plus much more secure! 

Thank you for taking the time to read this. I'm a security and privacy conscious individual, not associated with Accrescent or GrapheneOS (though I use them both).

Xoxo Olivia

**Comment by ghost** [comment]
Would you mind considering publishing the app in Accrescent ? It would be huge to have ente in Accrescent. And cool for users to be able to install ente without Google Play Store.

**Comment by barbiehunter** [comment]
Any news about it?

I noticed to keep it updated with Obtanium as example it gets some buggy with the Photos app. Same as would be simply nice to have there.

**Comment by chaitanyarajasekhar** [comment]
please consider ente auth & photos to be on accrescent store because of split apk support, developer signed keys, unattended updates. accrescent app is on grapheneos app store. It is also shown as a way for some apps to be downloaded on privacyguides.org

**Comment by ghost** [comment]
I'd love to see both ente photos and ente auth on Accrescent as well. Accrescent is already an option on ente's [privacypack](https://github.com/ente-io/privacypack) [website](https://privacypack.org/create), so why not release ente's apps there?

**Comment by ghost** [comment]
Any news yet? Really hoping that Accrescent can get some 2FA and cloud storage service apps which both Ente Auth and Photos can be a great starting point :).

**Reply by JaByteOS** [reply]
Honestly Accrescent has moved so slowly with their development, I've personally stopped using it. I now use obtainium to replace F-Droid and Accrescent. And unlike both it's highly secure (unlike F-Droid) and works with hundreds of thousands of apps (unlike Accrescent). Free, easy, amazing. I highly recommend it to everyone, and I no longer have to request developers for my favorite apps to do anything! Everyone wins!

**Reply by ghost** [reply]
> Honestly Accrescent has moved so slowly with their development, I've personally stopped using it. 

The "slow" development of Accrescent is mainly due to the fact that they are still a new project with limited resources and are more focused on developing backend feature for future developers and users. But with the recent changes and an successful attempt of calling for community support, the development has been faster than before.

> I now use obtainium to replace F-Droid and Accrescent. And unlike both it's highly secure (unlike F-Droid) and works with hundreds of thousands of apps (unlike Accrescent).

Obtainium is a sideloading tool. It can't replace a secure and functional app store for everything wise and isn't even that user friendly to began with. 

>Free, easy, amazing.

Accrescent is also free and easier than Obtainium to use, not sure whats your point in that.

> I highly recommend it to everyone, and I no longer have to request developers for my favorite apps to do anything! Everyone wins!

Just like other starting project, it just can't be supported right away and thats also applied to Accrescent. Devs need to be notified about it and it potentials in order to made up a decision if it worth it to support Accrescent or not. Accrescent might not be idea to support at the moment due to being whitelist only (they don't have the resource to handle every apps request but they can once they are ready) but with what Accrescent currently can and could offer in the future it would be an idea options for everyone.

**Comment by seathebeauty** [comment]
Hello @enteio, just chiming in as well in support of having Ente Photos and Ente Auth in the Accrescent app store. 

They are still small but have a mission aligned with yours and I'm sure the support of a mainstream app (as Ente.io is becoming) would be very beneficial for them and for you as sign of your commitment to free, open, private, and secure software. 

Cheers!
